core/src/main/java/hudson/TcpSlaveAgentListener.java

```diff
@@ -201,8 +201,6 @@ public ConnectionHandler(Socket s) {
         @Override
         public void run() {
             try {
-                LOGGER.log(Level.INFO, "Accepted connection #{0} from {1}", new Object[]{id,s.getRemoteSocketAddress()});
```

Comment:
Note: will no longer print anything for `GET` diagnostic requests, or for malformed requests or those requesting a nonexistent protocol. But probably that is for the best anyway.